server/recording_api.go

```diff
@@ -185,29 +191,23 @@ func (p *Plugin) handleRecordingStopAction(state *channelState, callID string) (
 		}
 	}()
 
-	// Sending the event prior to making the API call to the job service
-	// since it could take a few seconds to complete but we want clients
-	// to get their local state updated as soon as it changes on the server.
-	p.publishWebSocketEvent(wsEventCallRecordingState, map[string]interface{}{
-		"callID":   callID,
-		"recState": recState.getClientState().toMap(),
-	}, &model.WebsocketBroadcast{ChannelId: callID, ReliableClusterSend: true})
-
-	// We don't want to keep the lock while making the API call to the service since it
-	// could take a while to return.
-	p.unlockCall(callID)
```

Comment:
Unrelated to the panic but since we moved to the new job API we don't need to do this, the stop action is completely asynchronous and safe to be called under lock.